lib/ansible/plugins/callback/unixy.py

```diff
@@ -131,6 +131,9 @@ def v2_runner_on_ok(self, result, msg="ok", display_color=C.COLOR_OK):
     def v2_runner_item_on_failed(self, result):
         self.v2_runner_on_failed(result)
 
+    def v2_runner_item_on_ok(self, result):
+        self.v2_runner_on_ok(result)
+
```

Comment:
no 'on_skipped' ?

Reply:
Touche. Support for skipped output has been added.